addon/content/new-tab-variation.js

```diff
@@ -67,12 +67,14 @@ class TrackingProtectionStudy {
       if (tpContent) {
         // if already on the page, just update the message
         const spanEle = tpContent.getElementsByTagName("div")[0];
+        // eslint-disable-next-line no-unsanitized/property
```

Comment:
Not sure if I should have left these as warnings/errors instead of obscuring them via `eslint-disable-next-line`.
From my minor research, these did in fact seem like they were HTML strings (with `<span>`s and whatnot), so it wouldn't be appropriate to try setting them as text strings. However, I was not able to adequately investigate if there was any possibility of injecting anything which could get written out to the DOM by a user.

Reply:
Actually, this may be the general gist of #90